File: 240807/007/007.cpp
#include <iostream>
#include <string>

using namespace std;

class hello {
public: 
    string code;
    char meet;
    int time;

    hello(string code, char meet, int time){
        this->code = code;
        this -> meet = meet;
        this -> time = time;

        cout << "secret code : "<<code<<'\n';
        cout << "meeting point : "<<meet<<'\n';
        cout << "time : "<<time<<'\n';    
    }

};

int main() {
    string code;
    char meet;
    int time;

    cin >> code >> meet >> time;
    hello a = hello(code, meet, time);

    return 0;
}

File: 241019/007/007.py
class secret :
    def __init__(self, code, place, time) :
        self.c = code
        self.p = place
        self.t = time

ar = input().split()
ar[2] = int(ar[2])

s = secret(ar[0], ar[1], ar[2])

print("secret code :", s.c)
print("meeting point :", s.p)
print("time :", s.t)